server.js

```diff
@@ -1,7 +1,8 @@
 require('dotenv').config()
 const express = require ('express');
 const sequelize = require('./db')
-const PORT = process.env.PORT || 5000;
+const models = require('./models/relationships')
```

Comment:
`relationships` is no longer needed, everything is defined in index file now

Reply:
fixed